MobileWallet/Common/Theme.swift

```diff
@@ -86,6 +86,13 @@ struct Colors: Loopable {
     let creatingWalletEmojisLabelBackground = UIColor(named: "CreatingWalletEmojisLabel")
     let creatingWalletEmojisSeparator = UIColor(named: "CreatingWalletEmojiSeparator")
 
+    // Profile
+    let profileTitleTextColor = UIColor(named: "ProfileTitleTextBlack")
+    let profileCopyEmojiColor = UIColor(named: "ProfileCopyEmojiButton")
```

Comment:
Now that you're using `TextButton`, this color and font can be removed from the theme and assets